cli/tauri.js/bin/tauri.js

```diff
@@ -22,53 +26,67 @@ const tauri = function (command) {
     if (process.argv && !process.env.test) {
       process.argv.splice(0, 3)
     }
-    runOnRustCli(command, process.argv || []).promise.then(() => {
+    runOnRustCli(
+      command,
+      (process.argv || []).filter((v) => v !== '--no-update-notifier')
+    ).promise.then(() => {
       if (command === 'init') {
         const {
           installDependencies
         } = require('../dist/api/dependency-manager')
         return installDependencies()
       }
     })
-    return
-  }
+  } else {
+    if (
+      !command ||
+      command === '-h' ||
+      command === '--help' ||
+      command === 'help'
+    ) {
+      console.log(chalk.cyan(figlet.textSync('Tauri')))
```

Comment:
Why don't we just put the real display version of figlet here instead of pulling in another dependency?

Reply:
yeah, that works as well!